packages/web-components/src/_docs/concepts/introduction.stories.mdx

```diff
@@ -0,0 +1,26 @@
+import { Meta } from '@storybook/addon-docs';
+import pkg from '../../../package.json';
+
+<Meta title="Concepts/Introduction" />
+
+<h2>
+  Fluent UI Web Components <span>v{pkg.version}</span>
+</h2>
+
+⚠️ **While in Beta, breaking changes may be made based on feedback from users.** This space is useful for testing new components whose APIs might change before final release.
```

Comment:
```suggestion
```

Reply:
We're remaining in beta, so this is still valid.